docs/deploy-rbd.md

```diff
@@ -35,12 +35,12 @@ make image-cephcsi
 | `--instanceid`        | "default"             | Unique ID distinguishing this instance of Ceph CSI among other instances, when sharing Ceph clusters across CSI instances for provisioning                                                                                                                                           |
 | `--metadatastorage`   | _empty_               | Points to where legacy (1.0.0 or older plugin versions) metadata about provisioned volumes are kept, as file or in as k8s configmap (`node` or `k8s_configmap` respectively)                                                                                                         |
 | `--pidlimit`          | _0_                   | Configure the PID limit in cgroups. The container runtime can restrict the number of processes/tasks which can cause problems while provisioning (or deleting) a large number of volumes. A value of `-1` configures the limit to the maximum, `0` does not configure limits at all. |
-| `--metricsport`       | `8080`                | TCP port for liveness/grpc metrics requests                                                                                                                                                                                                                                          |
+| `--metricsport`       | `8080`                | TCP port for liveness metrics requests                                                                                                                                                                                                                                               |
 | `--metricspath`       | `"/metrics"`          | Path of prometheus endpoint where metrics will be available                                                                                                                                                                                                                          |
-| `--enablegrpcmetrics` | `false`               | Enable grpc metrics collection  and start prometheus server                                                                                                                                                                                                                          |
+| `--enablegrpcmetrics` | `false`               | [Deprecated] Enable grpc metrics collection  and start prometheus server                                                                                                                                                                                                             |
```

Comment:
Same here, not deprecated but dropped.

Reply:
I have added a deprecated message to command line argument? is there any otherway to deprecate a CLI argument?

Reply:
If something gets deprecated, there normally is a warning, but functionality remains the same. Upon removal, the command line argument may stay around, but with an error about the removed functionality.

Reply:
I removed the functionality and marked command-line argument as deprecated and if the command line is passed am logging it as a warning.

do you want me to add back the functionality?

Reply:
Either move back the functionality, or do not call it 'deprecated'

Reply:
reverted back the functionality @nixpanic PTAL